### PR TITLE
Add pst penalty cost parameter

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Enhancement of a parameter that was hardcoded to put it in configuration file

**What is the current behavior?** *(You can also link to an open issue here)*
Currently pst taps movement has a penalty cost in the objective function which is hardcoded.

**What is the new behavior (if this is a feature change)?**
The new behavior is to load this penalty cost in the configuration. Default value is 0.5.

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*
No breaking changes because a default value is set.

### CODE REVIEW

File: ra-optimisation/linear-rao/src/test/java/com/farao_community/farao/linear_rao/config/LinearRaoParametersTest.java [hunk 1]
```diff
@@ -61,4 +61,10 @@ public void setPstSensitivityThreshold() {
         raoParameters.setPstSensitivityThreshold(5.0);
         assertEquals(5.0, raoParameters.getPstSensitivityThreshold(), DOUBLE_TOLERANCE);
     }
+
+    @Test
+    public void setPstPenaltyCost() {
+        raoParameters.setPstPenaltyCost(0.5);
```
Comment (murgeyseb) [inline ra-optimisation/linear-rao/src/test/java/com/farao_community/farao/linear_rao/config/LinearRaoParametersTest.java:67]: Try the set method with a value different than default one

File: ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/config/LinearRaoParameters.java [hunk 2]
```diff
@@ -23,13 +23,15 @@
     static final int DEFAULT_MAX_NUMBER_OF_ITERATIONS = 10;
     static final boolean DEFAULT_SECURITY_ANALYSIS_WITHOUT_RAO = false;
     static final double DEFAULT_PST_SENSITIVITY_THRESHOLD = 0.0;
+    static final double DEFAULT_PST_PENALTY_COST = 0.5;
```
Comment (murgeyseb) [inline ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/config/LinearRaoParameters.java:26]: Why did you change default value (which was 1.0 previously) ?

Comment (TheMaskedTurtle) [reply ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/config/LinearRaoParameters.java]: it is specified to be set at 0.01. I previously put 0.5 to not break cucumber tests. Now I respect the specification so I will have another PR in cucumber tests

File: ra-optimisation/linear-rao/src/test/java/com/farao_community/farao/linear_rao/config/LinearRaoParametersTest.java (hunk 1 above)
Comment (TheMaskedTurtle) [reply ra-optimisation/linear-rao/src/test/java/com/farao_community/farao/linear_rao/config/LinearRaoParametersTest.java:67]: now default value is different :smile:
